docs/modules/servers/partials/operate/webadmin.adoc

```diff
@@ -94,6 +94,14 @@ Response codes:
 services can still be used.
 * 503: At least one check have answered with a Unhealthy status
 
+Additional query parameters are supported:
```

Comment:
Please add doc to `manage-webadmin.md` too.

BTW please complement https://github.com/apache/james-project/pull/2399/files doc also.